packages/core/src/utils/public.ts

```diff
@@ -290,3 +290,15 @@ export function getOsInfo() {
     });
   });
 }
+
+/**
+ * Shuffles array in place. ES6 version. This method is based on Fisher-Yates shuffle algorithm
+ * @param array An array containing the items.
+ */
+export function shuffle(array: any[]) {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [ array[i], array[j] ] = [ array[j], array[i] ];
+  }
+  return array;
```

Comment:
@rickycao-qy as you commented at https://github.com/alibaba/pipcook/pull/255#discussion_r443347354, we need to remove this return-statement to keep consistent.

Reply:
done